modules/client/test/unit/client-common/utils.test.ts

```diff
@@ -0,0 +1,78 @@
+import { computeProposalStatus, ProposalStatus } from "../../../src";
+
+describe("client-common utils", () => {
+  describe("computeProposalStatus", () => {
+    it("should return PENDING", () => {
+      const endDate = Date.now() / 1000;
+      const startDate = (Date.now() / 1000) + 500;
```

Comment:
Is it intentional to have a startDate greater than the end date? I think the protocol would not allow this